src/components/Docs/IncidentResponse.tsx

```diff
@@ -15,7 +15,7 @@ export default function IR() {
         <div className={styles.spaceBetween}>
           <div className={styles.moduleTitle}>
             <img src={`${baseUrl}img/icon-ir.svg`} />
-            <h1>Incident Response</h1>
+            <h1>Incident Response (COMING SOON) </h1>

```

Comment:
Not sure if this change is related to FME or not.

```suggestion
            <h1>Incident Response </h1>

```

Reply:
Thank you, nice catch.